src/sentry/integrations/jira/integration.py

```diff
@@ -74,6 +74,16 @@ def search_issues(self, query):
     def make_choices(self, x):
         return [(y['id'], y['name'] if 'name' in y else y['value']) for y in x] if x else []
 
+    def error_message_from_json(self, data):
+        message = ''
+        if data.get('errorMessages'):
+            message = ' '.join(data['errorMessages'])
+        if data.get('errors'):
+            if message:
+                message += ' '
+            message += ' '.join(['%s: %s' % (k, v) for k, v in data.get('errors').items()])
```

Comment:
I'm sure it was copied, but just joining with a space?

Reply:
gonna just leave this since yes, it was copied and don't want to spend time digging into what the format of this was